octopusdeploy/Chart.yaml

```diff
@@ -20,12 +20,4 @@ version: 0.1.16
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 2020.3.0
-
-dependencies:
-  - name: mssql-linux
-    version: 0.11.2
-    repository: https://kubernetes-charts.storage.googleapis.com/
-    condition: mssql-linux.enabled
-    tags:
-      - database
\ No newline at end of file
+appVersion: 2020.3.0
```

Comment:
I just learned we release a `latest` tagged image, so that would be the most convenient default.